src/View/Helper/FormHelper.php

```diff
@@ -1180,7 +1180,7 @@ protected function _inputContainerTemplate(array $options): string
         return $this->formatTemplate($inputContainerTemplate, [
             'content' => $options['content'],
             'error' => $options['error'],
-            'required' => $options['options']['required'] ? ' required' : '',
+            'required' => $options['options']['required'] ? 'is-required' : 'not-required',
```

Comment:
We can't change these default values either, as it will break existing applications. In the issue you opened it sounded more like you were hoping to add a template *for* the required class much like how `selectedClass` is implemented.